docs/setup-xdebug.md

```diff
@@ -0,0 +1,8 @@
+## Set up XDebug
+
```

Comment:
This part is misisng:

> Xdebug is shipped but disabled by default. It can be turned on by running:
> ```
> ./scripts/php-mod-config nextcloud xdebug.mode debug
> ```